tox.ini

```diff
@@ -3,7 +3,7 @@ envlist=py27,py33,py34,pypy,docs,self27,cov27
 
 # Default settings for py27, py33, py34 and pypy
 [testenv]
-deps=-r{toxinidir}/requirements.txt
+deps=-e {toxinidir}
```

Comment:
`-e` still here...have you tested it with tox? I haven't been able to make tox generally happy so trusting the committer until we get tox integrated with travis.

Reply:
For me tox run in a quite verbose way and is just failing in the doc env.
I commit something that makes it working better but tox is not totally ready I think.

Reply:
Is it closer to passing than on master, the same, or worse for you?  If this makes it better, I'll merge. Not going to release until I get tox passing on travis though.

Reply:
we've git exactky the same error in fact. 
I also think that one problem we have for documentation is that we use a very old version of sphinx + we do not add the "sphinx_rtd_theme" module.

Reply:
I think I found what I missed.